akka/shopping-cart-service-scala/src/test/scala/shopping/cart/ShoppingCartSimulation.scala

```diff
@@ -0,0 +1,137 @@
+package shopping.cart
+
+import java.util.UUID
+
+import akka.grpc.internal.Marshaller
+import com.github.phisgr.gatling.grpc.Predef._
+import com.github.phisgr.gatling.pb._
+import io.gatling.core.Predef._
+import io.grpc.{ ManagedChannelBuilder, MethodDescriptor }
+import shopping.cart.proto.ShoppingCartService.Serializers.{
+  AddItemRequestSerializer,
+  CartSerializer,
+  CheckoutRequestSerializer,
+  GetCartRequestSerializer,
+  UpdateItemRequestSerializer
+}
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+object ShoppingCartServiceMethodDescriptors {
+  val AddItem: MethodDescriptor[
```

Comment:
Those `MethodDescriptor`s are copies of the ones generated by Akka gRPC, but those aren't accessible. Copying here is far from ideal, and it is the quick-and-dirty option to make it work.